src/style.css

```diff
@@ -1 +1,81 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: bold;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  height: 60px;
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
u don't actually need these styles, just give paddings for header & it will adaptive